Mergin/ui/ui_status_dialog.ui

```diff
@@ -62,6 +62,13 @@
     </widget>
    </item>
    <item row="4" column="1">
+    <widget class="QPushButton" name="btn_reset_local_changes">
+     <property name="text">
+      <string>Reset Local Changes</string>
```

Comment:
I would shorten this to just "Reset Changes" - and add a tooltip what exactly it would do (e.g. "This will revert all changes in your local project directory.")

It may be good to also add an icon to the button, e.g. this one: https://tabler-icons.io/i/trash  ("View Changes" also has an icon)